hardhat.config.js

```diff
@@ -1,5 +1,6 @@
 require("dotenv").config();
 
+require("@nomiclabs/hardhat-truffle5");
```

Comment:
Why need hardhat truffle

Reply:
Other than that LGMT

Reply:
it is required for importing the `web3` package. Without this import I get errors like`Error: A web3 instance is not provided when not using truffle.`